test/train-sets/ref/inv_hash_load_model.invert.txt

```diff
@@ -1,4 +1,4 @@
-Version 8.9.0
```

Comment:
Are you planning on checking in a rev here?

Reply:
yes